pkgs/development/python-modules/cupy/default.nix

```diff
@@ -5,6 +5,10 @@
 , addOpenGLRunpath
 }:
 
+assert cudnn.cudatoolkit == cudatoolkit;
+assert cutensor.cudatoolkit == cudatoolkit;
+assert nccl.cudatoolkit == cudatoolkit;
```

Comment:
It was discovered [here](https://github.com/NixOS/nixpkgs/pull/166533#issuecomment-1084983987) that cupy was accidentally pulling in multiple cudatoolkit versions. These asserts should prevent that issue going forward.